neuror/cli.py

```diff
@@ -63,15 +64,21 @@ def repair():
               help='Where to save the plot')
 @click.option('-a', '--axon-donor', multiple=True,
               help='A morphology that provides a reference axon')
-@click.option('--plane',
+@click.option('--cut-file',
```

Comment:
is that ok for you to break the compat with older versions ?

Reply:
I think so. There is no other users.
Pinging @Stannislav  just in case.

Reply:
Fine with me!